### ISSUE TITLE
Not always able to add elements to rooms

### ISSUE THREAD

**Issue by caldien** [JulienNigon/SpellcasterUniversity-Bug-Report#1177]
With bigger university, sometimes i am unable to place things on walls/cauldrons/others in rooms and need to save and restart save to be able to do this (just had this issue with Window to another world and Rital components - after launching from save it works again).

**Comment by Sterla1** [comment]
You can override the issue by zooming a bit more and you can then place your objects ! But I have the same issue as well on Mac !

**Comment by caldien** [comment]
Yes, i managed to sole it by maximally zooming in the rooms (by double click), if i do that it kinda worked so far, however it is a bit annoying.

I am playing on Windows 10, just in case.

**Comment by Neelan22** [comment]
Yes it's a problem that has already been brought to our attention. We are thinking about how to improve it.
